lib/auth/trustedcluster.go

```diff
@@ -0,0 +1,94 @@
+package auth
+
+import (
+	"bytes"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/gravitational/teleport"
+
+	"github.com/gravitational/trace"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+type ValidateTrustedClusterResponse struct {
```

Comment:
I think this is a better way to structure request/response:

https://github.com/gravitational/gravity/blob/master/lib/ops/ops.go#L118

Notice `ToRaw` and `FromRaw` methods that encapsulate marshalling/unmarshalling